org.OpenT2T.Sample.SuperPopular.Thermostat/Wink Thermostat/v2/org.OpenT2T.Sample.SuperPopular.Thermostat-2.0.xml

```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!DOCTYPE node PUBLIC "-//freedesktop//DTD D-BUS Object Introspection 1.0//EN" "http://standards.freedesktop.org/dbus/introspect-1.0.dtd"[]>
+<node>
+  <interface name="org.OpenT2T.Sample.SuperPopular.Thermostat">
+    <description>OpenT2T SuperPopular Thermostat interface</description>
+
+    <property name="PowerStatus" type="s" access="readwrite">
```

Comment:
Oh, nice, I didn't realize ES2015 supported that property syntax, I thought it was just a TypeScript thing.

However, I think we should allow developers to follow both JavaScript and AllJoyn naming guidelines. JavaScript property names should be camelCase. The DeviceAccessor can do the case conversion, as it already does in other situations.

---
In reply to: [73775104](https://github.com/openT2T/translators/pull/37#discussion_r73775104) [](ancestors = 73775104,73772265,73771185,73768119)